File: CS/Scaffolding.CodeFirst/Data/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Scaffolding.CodeFirst.Model {
    public class Employee {
        public int EmployeeID { get; set; }
        public string Name { get; set; }
        public int DepartmentID { get; set; }
        public virtual Department Department { get; set; }
    }
}

File: VB/Scaffolding.CodeFirst/Data/Employee.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations

Namespace Scaffolding.CodeFirst.Model
	Public Class Employee
		Public Property EmployeeID() As Integer
		Public Property Name() As String
		Public Property DepartmentID() As Integer
		Public Overridable Property Department() As Department
	End Class
End Namespace